sway/commands.c

```diff
@@ -1126,21 +1126,21 @@ static struct cmd_results *cmd_bar(int argc, char **argv) {
 		return cmd_results_new(CMD_FAILURE, "bar", "Can only be used in config file.");
 	}
 
-	// Create new bar from default bar config
+	// Create new bar with default values
 	struct bar_config *bar = NULL;
-	bar = malloc(sizeof*bar);
-	bar->mode = strdup(config->bar.mode);
-	bar->hidden_state = strdup(config->bar.hidden_state);
-	bar->modifier = config->bar.modifier;
-	bar->position = config->bar.position;
+	bar = malloc(sizeof(struct bar_config));
+	bar->mode = strdup("dock");
+	bar->hidden_state = strdup("hide");
+	bar->modifier = 0;
+	bar->position = DESKTOP_SHELL_PANEL_POSITION_BOTTOM;
 	bar->bindings = create_list();
-	bar->status_command = strdup(config->bar.status_command);
-	bar->font = strdup(config->bar.font);
-	bar->bar_height = config->bar.bar_height;
-	bar->workspace_buttons = config->bar.workspace_buttons;
-	bar->strip_workspace_numbers = config->bar.strip_workspace_numbers;
-	bar->binding_mode_indicator = config->bar.binding_mode_indicator;
-	bar->tray_padding = config->bar.tray_padding;
+	bar->status_command = strdup("while :; do date +'%Y-%m-%d %l:%M:%S %p' && sleep 1; done");
+	bar->font = strdup("monospace 10");
+	bar->bar_height = -1;
+	bar->workspace_buttons = true;
+	bar->strip_workspace_numbers = false;
+	bar->binding_mode_indicator = true;
+	bar->tray_padding = 2;
```

Comment:
Yeah I will move it to a separate function.

Shouldn't it be possible to not define any bars?